app/assets/javascripts/questions.js

```diff
@@ -10,4 +10,16 @@ $(document).ready(function(q){
       $('.other-question').removeClass('hidden');
     });
 
+// Runs when the page loads and disables the submit button. Checks that both the answer and question text boxes are blank
+    if($('#question_text').val() || $('#question_answers_attributes_0_text').val() ==  "") 
+   $('#submit_question').attr('disabled', true);
+
+// Every time a key is pressed, this code runs and checks the contents of the answer and question text boxes. If both are full then the submit button is enabled 
+  $('#question_text, #question_answers_attributes_0_text').keyup(function(){
+    if($('#question_text').val() && $('#question_answers_attributes_0_text').val() !=  "") 
+         $('#submit_question').attr('disabled', false);    
+    else
+         $('#submit_question').attr('disabled', true);   
+  });
+
 });
```

Comment:
This little symbol indicates you should add a newline at the end of the file (this is important when using Unix utilities to sort, print, combine etc. text files, and is also helpful for diffs). You can mouse over the symbol to see Github's friendly message.